test/package.spec.ts

```diff
@@ -2,20 +2,50 @@
  * Copyright (c) 2019 SAP SE or an SAP affiliate company. All rights reserved.
  */
 
+import * as fs from 'fs-extra';
+import * as path from 'path';
+import Package from '../src/commands/package';
+import { getCleanProjectDir, getTestOutputDir } from './test-utils';
+
+const testOutputDir = getTestOutputDir(__filename);
+const nestAppDir = path.resolve('test', 'nest');
+
 describe('Package', () => {
-  it('should copy files correctly without parameters', () => {
-    expect(true).toBeTruthy();
+  afterAll(() => {
+    // fs.removeSync(testOutputDir);
   });
 
-  it('should copy files correctly with custom globs', () => {
-    expect(true).toBeTruthy();
+  it('should copy files correctly without parameters', async () => {
+    const projectDir = getCleanProjectDir(testOutputDir, 'no-params');
+    fs.copySync(nestAppDir, projectDir, { recursive: true });
+    await Package.run([projectDir, '--skipInstall']);
+
+    expect(fs.readdirSync(path.resolve(projectDir, 'deployment'))).toIncludeAllMembers(['package.json', 'package-lock.json']);
```

Comment:
From my understanding this tests the default value for `--include`. Shouldn't this test assert then that all files included by default are copied (i.e. package.json, package-lock.json, index.js, dist/**/*)?

Reply:
for the default nest.js app does not have an `index.js` and the `dist/**` only appear after compiling (which I dont test here). That's why I implemented it this way, but I am open for improvement suggestions.

Reply:
I see. Is there something like `toIncludeAndOnlyIncludeAllMembers`? =P